Formula/liblo.rb

```diff
@@ -36,4 +35,21 @@ def install
 
     system "make", "install"
   end
+
+  test do
+    File.write(
+      "lo_version.c",
+      "#include <stdio.h>
+      #include \"lo/lo.h\"
+      int main() {
+        char version[6];
+        lo_version(version, 6, 0, 0, 0, 0, 0, 0, 0);
+        printf(\"%s\", version);
+        return 0;
+      }",
+    )
+    system ENV.cc, "lo_version.c", "-I#{include}", "-L#{lib}", "-llo", "-o", "lo_version"
+    lo_version = `./lo_version`
+    assert_equal lo_version, "0.31"
```

Comment:
Can you use version.to_str here instead of a hard-coded 0.31

Reply:
Updated in 40b0806946735ca15adebaa02d87190c70353a2d